### PR TITLE
support scala-native

### PR DESCRIPTION
**Pull request by xuwei-k**
/cc @densh @MasseGuillaume

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,5 +1,11 @@
 scalacOptions += "-deprecation"
 
+addSbtPlugin("org.scala-native" % "sbt-crossproject" % "0.1.0")
```
Comment (MasseGuillaume) [inline project/plugins.sbt]: sbt-scala-native depends on sbt-crossproject you can remove this line if you want to.

**Comment by MasseGuillaume** [comment]
@xuwei-k really good. You can run the sbt task [nativeMissingDependencies](https://github.com/scala-native/scala-native/blob/master/sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala#L40) to see if anything is missing at link time.

File: project/plugins.sbt (hunk 1 above)
Comment (xuwei-k) [inline project/plugins.sbt]: Thanks. removed 😃

**Comment by xuwei-k** [comment]
I will remove `java.sql` instances https://github.com/scalaz/scalaz/pull/1324

**Pull request merged**